diesel/src/expression/expression_methods/sort_expression_methods.rs

```diff
@@ -0,0 +1,44 @@
+use expression::predicates::*;
+
+#[cfg(feature = "postgres")]
+pub trait Ordering : Sized {
+    /// Specify that nulls should come before other values in this ordering.
+    /// Normally, nulls come last when sorting in ascending order and first
+    /// when sorting in descending order.
```

Comment:
Could you add basically the same example as below here as well? (It's really nice to use doc tests for this, btw.)